content/_code-samples/escrow/js/finish-escrow.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+if (typeof module !== "undefined") {
+  // Use var here because const/let are block-scoped to the if statement.
+  var xrpl = require('xrpl')
+}
+
+const main = async () => {
+  try {
+    // Connect -------------------------------------------------------------------
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233');
+    await client.connect();
+    
+    // Get credentials from the Testnet Faucet ------------------------------------
+    console.log("Requesting an address from the Testnet faucet...");
+    const { wallet } = await client.fundWallet();
```

Comment:
If you're going to use a random account, it's best to create the escrow, and then finish it. Otherwise you end up showing an error instead of the ledger working as intended. Alternatively you can have the user provide the account with an escrow to finish, in which case there should be clear instructions for how to create an escrow, and get the information required to set up this bit of code. (Ex. "Run `node create-escrow.js` then copy the displayed secret into this prompt...") 

(Note: If you do that, it'd also require being able to get the secret from the `create-escrow.js` code in order to properly put it into this snippet)

Reply:
(This applies to both `finish-escrow.js` and `cancel-escrow.js`)

Reply:
Updated as per the suggestions, both the files now have Prerequisites comments